downstairs/src/lib.rs

```diff
@@ -3416,9 +3417,12 @@ enum WrappedStream {
 /// tests, it can be useful to create volumes using older backends.
 #[derive(Copy, Clone, Default, Debug, PartialEq)]
 pub enum Backend {
-    #[default]
+    #[cfg(any(test, feature = "integration-tests"))]
```

Comment:
I wonder if there is some value in allowing `crucible-downstairs` to pick what backend it
wants at creation time?  I think there could be value in making "old" versions for various kinds
of tests and not hiding them behind a feature would allow that.

It's also not something to worry about for this PR though.  Why do I even bring it up then?
Good question.  For which I have no answer..